airgun/browser.py

```diff
@@ -115,16 +86,12 @@ def get_browser(self):
         """
         if self.provider == 'selenium':
             return self._get_selenium_browser()
-        elif self.provider == 'saucelabs':
-            return self._get_saucelabs_browser()
-        elif self.provider == 'docker':
-            return self._get_docker_browser()
         elif self.provider == 'remote':
             return self._get_remote_browser()
         else:
             raise ValueError(
                 f'"{self.provider}" browser is not supported. '
-                f'Please use one of {("selenium", "saucelabs", "docker", "remote")}'
+                f'Please use one of {("selenium", "remote")}'
             )
 
     def post_init(self):
```

Comment:
Not sure what the purpose of this after removing everything.  Should it also be removed?

Reply:
Yes we can add later once we add that support in webkaifuku

Reply:
I'd just drop the whole function for now, unless you're going to put some statements in it.

IIRC doing anything after starting the browser instance is not something Ronny wanted included in the kaifuku design. I wouldn't assume that if such support was added, it would be done via a function called `post_init`.